src/hotspot/share/classfile/vmIntrinsics.cpp

```diff
@@ -488,6 +488,7 @@ bool vmIntrinsics::disabled_by_jvm_flags(vmIntrinsics::ID id) {
   case vmIntrinsics::_ghash_processBlocks:
     if (!UseGHASHIntrinsics) return true;
     break;
+  case vmIntrinsics::_base64_decodeBlock:
```

Comment:
I'd prefer to use consistent order. You have inserted decode after encode at other places.

Reply:
I agree.  Will fix.